test/theproto3/Makefile

```diff
@@ -31,4 +31,5 @@ regenerate:
 	cat maps.proto >> theproto3.proto
 	cat footer.proto >> theproto3.proto
 	go install github.com/gogo/protobuf/protoc-gen-combo
-	protoc-gen-combo --version="3.0.0" --proto_path=../../../../../:../../protobuf/:. theproto3.proto
\ No newline at end of file
+	protoc-gen-combo --version="3.0.0" --proto_path=../../../../../:../../protobuf/:. theproto3.proto
+	find combos -type d -not -name combos -exec cp proto3_test.go.in {}/proto3_test.go \;
```

Comment:
Ok this looks cool and is the correct way to do it, but not everyone knows `find` that well.
Could we rather just "manually" copy the proto3_test.go.in to each folder?